### PR TITLE
added error messages

### PR DESCRIPTION
**Pull request by HeyBail**

### CODE REVIEW

File: src/views/AddCourse.vue [hunk 1]
```diff
@@ -15,6 +15,7 @@ const course = ref({
   description: ""
 });
 const message = ref("Enter data and click save");
+const errors = ref({});
 
 function addCourse() {
   CourseServices.addCourse(course.value)
```
Comment (JennaStilling) [inline src/views/AddCourse.vue:21]: I would clear any existing errors before updating the error message(s):
image
image
When I received an initial error, I fixed the error but "accidentally" messed up in another field. When hitting add again, I still had all the old errors, and the field in which the error happened was unclear at first glance

**Review by JennaStilling — COMMENTED** [review]
See add course comment, same applies for edit course

File: src/views/AddCourse.vue (hunk 1 above)
Comment (HeyBail) [reply src/views/AddCourse.vue:21]: fixed

Comment (JennaStilling) [reply src/views/AddCourse.vue:21]: Looks good

**Pull request merged**